src/main/java/Aples.java

```diff
@@ -0,0 +1,98 @@
+        list[itemIndex] = new Todo(description);
+        Print.echoTask(list[itemIndex], itemIndex + 1);
+    }
+
+    public static void handleDeadlineTask(String line, Task[] list, int itemIndex) {
+        if (line.contains("/by")) {
+            int byPosition = line.indexOf("/by");
+            String description = line.substring(9, byPosition);
+            String deadline = line.substring(byPosition + 4);
+            list[itemIndex] = new Deadline(description, deadline);
+            Print.echoTask(list[itemIndex], itemIndex + 1);
+        } else {
+            System.out.println("-".repeat(Print.LINE_DASH_LENGTH));
+            System.out.println("Please key in a Deadline task with the keyword '/by'");
+            System.out.println("-".repeat(Print.LINE_DASH_LENGTH));
+            String newLine;
+            Scanner in = new Scanner(System.in);
+            newLine = in.nextLine();
+            handleDeadlineTask(newLine, list, itemIndex);
+        }
+    }
+
+    public static void handleEventTask(String line, Task[] list, int itemIndex) {
+        if (line.contains("/from") && line.contains("/to")) {
+            int fromPosition = line.indexOf("/from");
+            int toPosition = line.indexOf("/to");
+            String description = line.substring(6, fromPosition);
+            String startTime = line.substring(fromPosition + 6, toPosition);
+            String endTime = line.substring(toPosition + 4);
+            list[itemIndex] = new Event(description, startTime, endTime);
+            Print.echoTask(list[itemIndex], itemIndex + 1);
+        } else {
+            System.out.println("-".repeat(Print.LINE_DASH_LENGTH));
+            System.out.println("Please key in an Event task with both the keywords '/from' and '/to'");
+            System.out.println("-".repeat(Print.LINE_DASH_LENGTH));
+            String newLine;
+            Scanner in = new Scanner(System.in);
+            newLine = in.nextLine();
+            handleEventTask(newLine, list, itemIndex);
+        }
+    }
+
+    public static void main(String[] args) {
+        String logo =
+                "                                               \n" +
+                " █████╗ ██████╗ ██╗     ███████╗███████╗       \n" +
+                "██╔══██╗██╔══██╗██║     ██╔════╝██╔════╝       \n" +
+                "███████║██████╔╝██║     █████╗  ███████╗       \n" +
+                "██╔══██║██╔═══╝ ██║     ██╔══╝  ╚════██║       \n" +
+                "██║  ██║██║     ███████╗███████╗███████║       \n" +
+                "╚═╝  ╚═╝╚═╝     ╚══════╝╚══════╝╚══════╝       \n" +
+                "    Aples chatbot at your service              \n";
+        System.out.println(logo);
+        Task[] list = new Task[MAX_ENTRIES]; //make an array of class type: Task
+        int itemIndex = 0; //index of the item to be added to the list
+        Print.printGreeting();
+
+        while (true) {
```

Comment:
I think a boolean can be created and pass into the while loop instead; while (true) loops are generally not recommended as it may cause infinite loops! :0
```suggestion
        while (true) {
```

Reply:
it would be better to provide a clearer condition as to when to exit the loop instead of just using while(true)